install.sh

```diff
@@ -47,10 +47,10 @@ get_password () {
   local password
   local prompt="${1}: "
   local retry_prompt="
-The password cannot be empty!
+The password must contain minimum of 8 characters!
 ${prompt}"
   read -rs -p "${prompt}" password
-  until [ -n "${password}" ]; do
+  until [ ${#password} -ge 8 ]; do
```

Comment:
We can't demand a minimum password length when the user is connecting to a preexisting DB